src/pages/index.tsx

```diff
@@ -168,7 +253,13 @@ const Home: React.FC = () => {
           </div>
         </div>
       </header>
+      <br></br>
       <main>
+        <br></br>
+        <br></br>
+        <center>
+          <img src="img/logos/Browsers.png" width="40%"></img>
```

Comment:
nit: have lowercase file names

Reply:
@mxschmitt: these are the existing files: Adobe2.png  Bing.png  DHotstar.jpg  ING.png  MUI.png  Outlook.png  ReactNavigation.png  VSCode.png.